R/cmapplot.R

```diff
@@ -44,7 +44,8 @@ if (.Platform$OS.type == "windows") {
       note = list(family="Whitney Book", face="plain", size=11),
       title = list(family="Whitney Semibold", face="plain", size=17),
       axis = list(family="Whitney Book", face="plain", size=14),
-      label = list(family="Whitney Semibold", face="plain", size=14)
+      label = list(family="Whitney Semibold", face="plain", size=5)
```

Comment:
These are going to need to be reconfigured to match @matthewstern's font restructuring once #72 is merged.

Reply:
Yup, good flag.